### ISSUE TITLE
GPU load was not same as windows taskmgr

### ISSUE THREAD

**Issue by capric8416** [winsiderss/systeminformer#1458]
### Brief description of your issue

**When I play multiple videos at the same time, System Informer show GPU load was not same with windows taskmgr.**

![image](https://user-images.githubusercontent.com/7124338/199187135-c97bd7a4-5937-4cc1-8e68-f96c656e218f.png)

I found a pull request from T-Troll/alienfx-tools, it looks fixed at
https://github.com/T-Troll/alienfx-tools/pull/227

### Steps to reproduce (optional)

1. Open windows taskmgr, swith to performance tab, click GPU 0.
2. Play 2 videos with mpv, vlc, or other video players.
3. Open System Informer, show GPU Utilization percentage.
4. You will find System Informer's GPU load was about half of windows taskmgr

### Environment (optional)

```shell
System Informater version: c19d69317a8eedcce773eb7317462eac8dfebf66
Windows Version: Windows 10 21H2 LTSC
```

**Comment by dmex** [comment]
> group by engine type and sum, then find max

Thanks for figuring it out 👍 

Fixed in 4e9a23f78ea1454034dd362806f244cbe4c73eb7

**Comment by capric8416** [comment]
@dmex Thank you very much for the bug fix
